build.gradle

```diff
@@ -22,21 +42,49 @@ plugins {
 }
 
 
+apply from: 'gradle/formatting.gradle'
+apply plugin: 'maven-publish'
+apply plugin: 'opensearch.opensearchplugin'
+apply plugin: 'opensearch.testclusters'
+
+opensearchplugin {
+    name 'opensearch-sdk-java'
+    description 'OpenSearch Software Development Kit'
+    classname 'Sample Class'
```

Comment:
This class name should be the fully qualified class name of the main class, take Job Scheduler's [build.gradle](https://github.com/opensearch-project/job-scheduler/blob/ad33680eb6c2ea69ebc463cdbbee7e2a396d3e05/build.gradle#L58) for example